code/app/src/androidTest/java/com/cmput301w21t06/crowdfly/TrialActivityTest.java

```diff
@@ -56,42 +56,42 @@ public void tearDown() {
         FirebaseAuth.getInstance().signOut();
     }
 
-    @Test
```

Comment:
We should probably leave this active so that we know its failing and we need to fix it. Otherwise it'll go unnoticed.